lib/battle/BattleHex.h

```diff
@@ -8,7 +8,15 @@
  *
  */
 #pragma once
-#include "../GameConstants.h"
+
+namespace BattleSide
+{
+	enum

```

Comment:
It can be changed to `enum class BattleSide : ui8` everywhere, but then it will require casts to access BattleInfo::sides